include/grl/flatbuffer/KUKAiiwa.fbs

```diff
@@ -63,7 +63,9 @@ enum EControlMode : byte {
   POSITION_CONTROL_MODE=0,
   CART_IMP_CONTROL_MODE=1,
   JOINT_IMP_CONTROL_MODE=2,
-  NO_CONTROL=3
+  NO_CONTROL=3,
```

Comment:
I started going through things in more detail and I realized this enum change isn't an acceptable one. If you look in `FRI_Client_SDK_Cpp.zip` there is a file `FRIMessages.pb.h`. This contains the Enum `EControlMode` which matches the current master perfectly. 

I originally thought that KUKA may have added additional control modes but I now realize this was a misunderstanding of the design.